helm/sematic-server/values.yaml

```diff
@@ -8,6 +8,16 @@ aws:
   storage_bucket: my-s3-bucket # REPLACE ME
   bucket_region: my-s3-region # REPLACE ME
 
+gcp:
+  storage:
+    enabled: false
+    bucket:  my-bucket # REPLACE ME
+  service_account:
+    secret:
+      enabled: false
+      name: gcp-sa # REPLACE ME
```

Comment:
since we're now supporting both the secret injection and the service account workload identity patterns, you'll need a separate `service_account: name: xxx` field.  if `secret.enabled` is `false`, users still need the ability to specify the service account name.